hikari/impl/shard.py

```diff
@@ -666,7 +672,11 @@ def _dispatch(self, name: str, seq: int, data: data_binding.JSONObject) -> None:
             self._logger.info("shard has resumed [session:%s, seq:%s]", self._session_id, self._seq)
             self._handshake_completed.set()
 
-        self._event_consumer(self, name, data)
+        try:
+            self._dispatcher.consume_raw_event(self, name, data)
+
+        except LookupError:
+            self._logger.debug("ignoring unknown event %s:\n    %r", name, data)
```

Comment:
Unknown events should be handled in `consume_raw_event`

Reply:
Would that mean adding an unknown event handler or just having consume_raw_event be a black box where you can't tell if it actually dispatched an event or just ignore it at runtime after calling it without also calling get_raw_consumers afterwards?

Reply:
Hmm, true, good point

Reply:
Also random point but logging it here has the added bonus of telling you which shard the payload was received on